backend/corpora/parliament/euparl.py

```diff
@@ -193,15 +194,18 @@ def parse_graph_from_filename(self, filename: str) -> Graph:
     )
 
     speaker_country = FieldDefinition(
-        name='speaker_country',
-        display_name='Represented country',
-        description='The EU country the speaker represents',
+        name="speaker_country",
```

Comment:
Probably autoformatter deciding to go with double quotes?

Reply:
Yes, indeed.

Reply:
At present, `'` is more commonly used in the I-analyzer backend, so if your reason for using an autoformatter is consistency, it probably makes more sense to configure the formatter with single quotes.

Reply:
Yes, problematic is that for another project I'm using double quotes now. :-P

Reply:
The auto-formatter is "Black", and overall I'm quite happy with how it works. I will check if I can configure it otherwise.